src/iexamine.h

```diff
@@ -14,6 +14,7 @@ class game;
 class item;
 class player;
 class map;
+class tripoint;
```

Comment:
```
ccache clang++  -DRELEASE -DLOCALIZE -Os -Wall -Wextra   -Werror --std=c++11 -MMD -I/usr/include/lua5.2     -DLUA -I/usr/include/ncursesw -c src/action.cpp -o obj/action.o
[...]
In file included from src/action.cpp:7:
In file included from src/map.h:15:
In file included from src/mapdata.h:11:
src/iexamine.h:17:1: error: class 'tripoint' was previously declared as a struct [-Werror,-Wmismatched-tags]
class tripoint;
^
src/enums.h:248:8: note: previous use is here
struct tripoint : public JsonSerializer, public JsonDeserializer {
       ^
src/iexamine.h:17:1: note: did you mean struct here?
class tripoint;
^~~~~
struct
1 error generated.
make: *** [obj/action.o] Error 1
```
-- <http://ci.narc.ro/job/Cataclysm-Matrix/Graphics=Curses,Platform=Linux/3261/console>.

This is purely advisory, I'll get it fixed in a moment.